packages/fxa-payments-server/src/components/PaymentMethodHeader/en.ftl

```diff
@@ -2,5 +2,5 @@
 
 payment-method-header = Choose your payment method
 # This message is used to indicate the second step in a multi step process.
-payment-method-header-second-step = 2. { payment-method-header }
```

Comment:
Just for reference there was a small discussion regarding this string, [found here](https://github.com/mozilla/fxa-content-server-l10n/pull/540#discussion_r772116836).

Maybe a question for l10n, is it typically bad practice or not recommended to use a placeable like this?

Reply:
I'll close the PR then.